python/ray/actor.py

```diff
@@ -494,24 +539,42 @@ def _remote(self,
             if actor_method_cpu == 1:
                 actor_placement_resources = resources.copy()
                 actor_placement_resources["CPU"] += 1
-            function_signature = meta.method_signatures[function_name]
-            creation_args = signature.flatten_args(function_signature, args,
-                                                   kwargs)
+            if meta.is_cross_language:
+                if not worker.load_code_from_local:
+                    raise Exception("Cross language feature needs "
+                                    "--load-code-from-local to be set.")
+                if meta.language == Language.PYTHON:
+                    # To compatible with ray.signature.recover_args
+                    function_signature = signature.ANY_FUNCTION_SIGNATURE
+                    creation_args = signature.flatten_args(
+                        function_signature, args, kwargs)
+                else:
```

Comment:
could you add a comment that here we are calling non-python?

Reply:
OK, I have updated code.